Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -331,6 +333,33 @@ GDCMImageIO::Read(void * pointer)
     // WARNING: sizeof(Real World Value) != sizeof(Stored Pixel)
     len = len * outputpt.GetPixelSize() / pixeltype.GetPixelSize();
   }
+  /*
+   * YBR to RGB
+   * Currently (GDCM 3.0.2) processing for lossy Jpeg YBR_FULL_422
+   * is not fully clear. May be later the files will be converted
+   * internally in GDCM, in that case remove YBR_FULL_422 below.
+   *
+   * TODO: UINT16
+   *
+   */
+  if (m_NumberOfComponents == 3 &&
+      (pi == gdcm::PhotometricInterpretation::YBR_FULL || pi == gdcm::PhotometricInterpretation::YBR_FULL_422) &&
+      m_YBRtoRGB && (pixeltype == gdcm::PixelFormat::UINT8 || pixeltype == gdcm::PixelFormat::INT8) && (len % 3 == 0))
+  {
+    unsigned char * copy = reinterpret_cast<unsigned char *>(pointer);
+    for (size_t x = 0; x < len; x += 3)
+    {
+      const double Y = static_cast<double>(copy[x]);
```

Comment:
I would have used the gdcm filter YBR to RGB instead of copy/pasting code.

Reply:
Thank you. Sorry, change photometric interpretation from GDCM crashes with lossy Jpeg, s, previous closed PR and allocates memory. Sorry.